samza-sql-shell/conf/shell-defaults.conf

```diff
@@ -22,6 +22,8 @@
 # Example:
 shell.executor = org.apache.samza.sql.client.impl.SamzaExecutor
 samza.sql.output = compact
+# shell.commandhandler = com.linkedin.CustomeCommandHandler1
+# shell.commandhandler = com.linkedin.CustomeCommandHandler2
```

Comment:
We may prefer a value list to multiple commandhandler entries, as multiple entries could pervade the configuration file and become hard to notice.

Reply:
made it a comma separated list